Tables/External Tables.event

```diff
@@ -65,3 +75,14 @@ MountedClassTable:
 //Replace this with a list of your classes, mounted and unmounted.
 MountedUnmounted(VoidMaster, VoidMonster)
 EndMountUnmountedTable
+
+
+//Weapon Table
+NewManaketeAnimSet: //C9 = Already transformed Morva-dragon
```

Comment:
This works for Babby, as well as Ersatz, I assume?

Reply:
i believe bab was put in morva's class; not pre-transformed manakete